installer/cape2.sh

```diff
@@ -604,43 +741,44 @@ function install_yara() {
 }
 
 function install_mongo(){
-    echo "[+] Installing MongoDB"
-    # Mongo >=5 requires CPU AVX instruction support https://www.mongodb.com/docs/manual/administration/production-notes/#x86_64
-    if grep -q ' avx ' /proc/cpuinfo; then
-        MONGO_VERSION="6.0"
-    else
-        echo "[-] Mongo >= 5 is not supported"
-        MONGO_VERSION="4.4"
-    fi
-
-    sudo curl -fsSL "https://www.mongodb.org/static/pgp/server-${MONGO_VERSION}.asc" | sudo gpg --dearmor -o /etc/apt/keyrings/mongo.gpg --yes
-    echo "deb [signed-by=/etc/apt/keyrings/mongo.gpg arch=amd64] https://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/${MONGO_VERSION} multiverse" > /etc/apt/sources.list.d/mongodb.list
+	if ( "$mongo_enable" -ge 1 ); then
```

Comment:
Should be brackets:
`if [ "$mongo_enable" -ge 1 ]; then`

Reply:
Ohh! Derp! Thanks. Yeah, was just about to find that here as well as just getting ready to test this all via ansible. =^.^=

Reply:
added fix to a  PR in https://github.com/kevoreilly/CAPEv2/pull/1334